Runtime/UI/SafeArea.cs

```diff
@@ -65,14 +57,30 @@ private void OnDisable()
         /// <returns>True if the RectTransform was changed.</returns>
         private bool Apply()
         {
-            if (Canvas == null)
+            if (canvas == null)
                 return false;
 
-            // Force update if anchorMax is not zero because the RectTransform value will be incorrect if the RectTransform value is changed and undo.
-            if (RectTransform.anchorMax != Vector2.zero && m_isInitialized && Screen.safeArea == m_latestSafeArea)
-                return false;
+            if (m_isInitialized)
+            {
+                if (Application.isPlaying)
+                {
+                    // If this flag is set to false, we apply the safe area only at initialization.
+                    if (!checkSafeAreaSizeAtRuntime)
```

Comment:
I'd move this check directly into the beginning of the Update() method and just return if it's false

Reply:
Thanks, that's right. I fixed it!